pkg/apis/eksctl.io/v1alpha5/validation_test.go

```diff
@@ -708,6 +708,21 @@ var _ = Describe("ClusterConfig validation", func() {
 			})
 		})
 
+		Context("extraIPv6CIDRs", func() {
+			It("validates cirds", func() {
+				cfg.VPC.ExtraIPv6CIDRs = []string{"2002::1234:abcd:ffff:c0a8:101/64"}
+				err = cfg.ValidateVPCConfig()
+				Expect(err).ToNot(HaveOccurred())
+			})
+
+			When("extraCIDRs has an invalid cidr", func() {
```

Comment:
can you add one please with just an invalid block identifier?

Reply:
done